src/main/options-normalizer.js

```diff
@@ -18,41 +17,46 @@ const cliDescriptor = {
       : `${cliDescriptor.key(key)}=${value}`,
 };
 
-class FlagSchema extends vnopts.ChoiceSchema {
-  constructor({ name, flags }) {
-    super({ name, choices: flags });
-    this._flags = [...flags].sort();
-  }
-  preprocess(value, utils) {
-    if (
-      typeof value === "string" &&
-      value.length > 0 &&
-      !this._flags.includes(value)
-    ) {
-      const suggestion = this._flags.find((flag) => leven(flag, value) < 3);
-      if (suggestion) {
-        utils.logger.warn(
-          [
-            `Unknown flag ${chalk.yellow(utils.descriptor.value(value))},`,
-            `did you mean ${chalk.blue(utils.descriptor.value(suggestion))}?`,
-          ].join(" ")
-        );
-        return suggestion;
+const getFlagSchema = (colorsModule) =>
```

Comment:
?
```suggestion
// To prevent `chalk` module from being included in the `standalone.js` bundle,
// it will take that as an argument if needed.
const getFlagSchema = (colorsModule) =>
```